modules/prefill/README.rst

```diff
@@ -22,9 +22,9 @@ Example configuration is:
               }
         })
 
-This configuration downloads zone file from URL `https://www.internic.net/domain/root.zone` and imports it into cache every 86400 seconds (1 day). The HTTPS connection is authenticated using CA certificate from file `/etc/pki/tls/certs/ca-bundle.crt` and signed zone content is validated using DNSSEC.
+This configuration downloads the zone file from URL `https://www.internic.net/domain/root.zone` and imports it into the cache every 86400 seconds (1 day). The HTTPS connection is authenticated using a CA certificate from file `/etc/pki/tls/certs/ca-bundle.crt` and signed zone content is validated using DNSSEC.
 
-Root zone to import must be signed using DNSSEC and the resolver must have valid DNSSEC configuration.
+Root zones to be imported must be signed using DNSSEC and the resolver must have a valid DNSSEC configuration.
```

Comment:
"Root zones" sounds weird. Perhaps "The root zone" instead? All the other changes seem good to me.

Reply:
Yep. You're right about that. Changed it.